### ISSUE TITLE
`prefer-scoped-selector` doesn't report selector lists

### ISSUE THREAD

**Issue by fregante** [sindresorhus/eslint-plugin-unicorn#3256]
By the way, the rule currently does not report this:

```js
// ❌
el.querySelector('div .a, div b')!;
// ✅ but shouldn't
el.querySelector(':scope div .a, div b')!;
```

even though the second part of the selector is not covered by `:scope`. This is why I had suggested to avoid selector-related rules, they require parsing or else it will be tricked by these too:

```js
el.querySelector(':scope div:is([title*=", "], div b)')!;
```

Related:

- https://github.com/sindresorhus/eslint-plugin-unicorn/issues/3254

**Comment by fregante** [comment]
🥲 https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-scoped-selector.md#limitations

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/3259
